pandas/tests/indexes/datetimes/test_indexing.py

```diff
@@ -46,6 +46,15 @@ def test_where_tz(self):
         expected = i2
         tm.assert_index_equal(result, expected)
 
+    @pytest.mark.parametrize('tz', [None, 'UTC', 'US/Eastern'])
+    def test_insert_nat(self, tz):
+        # GH#16537, GH#18295 (test missing)
+        idx = pd.DatetimeIndex(['2017-01-01'], tz=tz)
+        expected = pd.DatetimeIndex(['NaT', '2017-01-01'], tz=tz)
+        for null in [None, np.nan, pd.NaT]:
```

Comment:
can you pull this null into the paramaterization?

Reply:
Sure, pushing momentarily.